js/index.js

```diff
@@ -14,9 +14,64 @@ function bindToForm(form) {
 	update()
 
 	form.addEventListener('change', update)
-	form.addEventListener('keyup', update)
+	form.addEventListener('keyup', function(e) {
+		handleDynamicForms(e, update)
+	})
 }
 
+function handleDynamicForms(e, next) {
+
+	function setCommuneFetchStatus(text) {
+		var label = document.querySelector('label[for="code_postal_entreprise"]')
+		label.innerHTML = text || ''
+
+		// Clear depcom_entreprise <select>
+		depcomElement.innerHTML = ''
+		depcomElement.setAttribute('hidden', '')
+	}
+	/*
+	code_postal_entreprise <input> value is used to dynamically
+	fetch a list of options for the the depcom_entreprise <select> element
+	*/
+	if (e.target.name && e.target.name === 'code_postal_entreprise') {
```

Comment:
Comparing `undefined` to `code_postal_entreprise` will yield `false`, no need to check for truthiness first.